packages/paste-core/components/form-pill-group/stories/index.stories.tsx

```diff
@@ -177,6 +179,157 @@ export const FormPillTreeVariant = (): JSX.Element => {
+              selected={selected}
+              variant={index > 2 ? "error" : "default"}
+              onDismiss={dismissable ? () => {} : undefined}
+              disabled={disabled}
+            >
+              {index % 3 === 2 ? (
+                <Avatar
+                  size={size === "large" ? "sizeIcon20" : "sizeIcon10"}
+                  name="avatar example"
+                  src="./avatars/avatar4.png"
+                />
+              ) : null}
+              {index % 3 === 1 ? (
+                <CalendarIcon decorative size={size === "large" ? "sizeIcon20" : "sizeIcon10"} />
+              ) : null}
+              {pill}
+            </FormPill>
+          ))}
+        </FormPillGroup>
+
+        <FormPillGroup {...pillState2} data-testid="form-pill-group-2" aria-label={ariaLabel} size={size}>
+          {PILL_NAMES.map((pill, index) => (
+            <FormPill
+              key={`${pill}-2`}
+              data-testid={`form-pill-${index}-02`}
+              {...pillState2}
+              selected={selected}
+              variant={index > 2 ? "error" : "default"}
+              onDismiss={dismissable ? () => {} : undefined}
+              disabled={disabled}
+            >
+              {pill}
+            </FormPill>
+          ))}
+        </FormPillGroup>
+      </Stack>
+    </Box>
+  );
+};
+
+PillStringOverflowVSComposed.storyName = "Pill String Overflow vs Composed";
+
+export const PillNarrowContainerDecorationsEnd: React.FC<
+  React.PropsWithChildren<{
+    selected?: boolean;
+    dismissable?: boolean;
+    disabled?: boolean;
+    ariaLabel?: string;
+    size?: FormPillGroupSizeVariant;
+  }>
+> = ({ selected = false, dismissable = true, disabled = false, ariaLabel = "Basic pills:", size }) => {
+  const pillState = useFormPillState();
+  const pillState2 = useFormPillState();
+
+  return (
+    <Box maxWidth="150px">
+      <FormPillGroup {...pillState} data-testid="form-pill-group" aria-label={ariaLabel} size={size}>
+        {PILL_NAMES.map((pill, index) => (
+          <FormPill
+            key={`${pill}-1`}
+            data-testid={`form-pill-${index}`}
+            {...pillState}
+            selected={selected}
+            variant={index > 2 ? "error" : "default"}
+            onDismiss={dismissable ? () => {} : undefined}
+            disabled={disabled}
+          >
+            {pill}
+            {index % 3 === 2 ? (
+              <Avatar
+                size={size === "large" ? "sizeIcon20" : "sizeIcon10"}
+                name="avatar example"
+                src="./avatars/avatar4.png"
+              />
+            ) : null}
+            {index % 3 === 1 ? <CalendarIcon decorative size={size === "large" ? "sizeIcon20" : "sizeIcon10"} /> : null}
+          </FormPill>
+        ))}
+      </FormPillGroup>
+    </Box>
+  );
+};
+
+PillNarrowContainerDecorationsEnd.storyName = "Pill Narrow Container Decorations End";
+
+export const PillNarrowContainerFormattedTextContent: React.FC<
```

Comment:
```suggestion
export const PillNarrowContainerFormattedTextContent: StoryFn<
```

could you update the stories in this file to use StoryFn (imported from storybook/react)?